src/views/Index.vue

```diff
@@ -1,10 +1,9 @@
 <script setup lang="ts">
-// This starter template is using Vue 3 <script setup> SFCs
-// Check out https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup
-import HelloWorld from '../components/HelloWorld.vue'
+  import Home from "../components/Home.vue"
 </script>
 
 <template>
-  <img alt="Vue logo" src="../assets/logo.png" />
-  <HelloWorld msg="Hello Vue 3 + TypeScript + Vite" />
-</template>
+  <div id="app">
```

Comment:
The div id for app is defined in index.html, so this causes a nested effect where Vue.js will attempt to render everything inside of both of them (potentially causing an infinite recursion).

I'd try to abandon IDs unless necessary and embrace the Vue.js way of doing things.